### ISSUE TITLE
Error when converting a directory containing .mp4 files

### ISSUE THREAD

**Issue by sfetzel** [Faster3ck/Converseen#138]
Converseen shuts down with the following error, when converting a directory containing a .mp4 file:
```
[NULL @ 0x564adce39a80] Requested output format 'rawvideo' is not a suitable output format
/tmp/magick-AD0XuhGvwoi6n9qMx5j_mJTbndVzajCp: Invalid argument
terminate called after throwing an instance of 'Magick::ErrorImage'
  what():  converseen: negative or zero image size `/media/myfile.mp4' @ error/resize.c/ResizeImage/3737
```
How to reproduce:

- Select a directory containing jpg and mp4 files
- Set "Convert to" to Don't convert.
- Set scale image to e.g. 20% of the width and 20% of the height.
- Start conversion.
- Application shuts down as soon as it processes the .mp4 file.

Expected outcome:
The application skips the .mp4 files.

**Comment by Faster3ck** [comment]
Hello, thank you very much for reporting the bug! I am currently working on fixing it.

**Comment by Faster3ck** [comment]
Hi, the problem has been fixed in [version 0.11.0.0](https://github.com/Faster3ck/Converseen/releases/tag/v0.11.0.0): Now, the program will skip unsupported files in write mode.

Thanks again for the bug report!
